pom.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright © 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.gravitee.policy</groupId>
+    <artifactId>gravitee-policy-mtls</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+
+    <name>Gravitee.io APIM - Policy - mTLS</name>
+    <description>Check client certificate against gateway trusted certificates, allowing only consumers with trusted certificates to access your APIs</description>
+
+    <parent>
+        <groupId>io.gravitee</groupId>
+        <artifactId>gravitee-parent</artifactId>
+        <version>22.0.2</version>
+    </parent>
+
+    <properties>
+        <gravitee-bom.version>8.1.0</gravitee-bom.version>
+        <gravitee-gateway-api.version>3.5.1</gravitee-gateway-api.version>
+        <gravitee-policy-api.version>1.11.0</gravitee-policy-api.version>
+        <gravitee-apim.version>4.4.3</gravitee-apim.version>
+
+        <maven-plugin-assembly.version>3.5.0</maven-plugin-assembly.version>
+        <maven-plugin-prettier.version>0.19</maven-plugin-prettier.version>
+        <maven-plugin-prettier.prettierjava.version>2.1.0</maven-plugin-prettier.prettierjava.version>
```

Comment:
I think these version are handled by the parent. You don't need to specify them anymore.